### ISSUE TITLE
No source and target

### ISSUE THREAD

**Issue by xindi-dumbledore** [jeremy-umuc/gexf-app#1]
Hi, very useful tool. However, I noticed that the edge table is missing source and target in the edge table in cytoscape. Could you tell me how to fix that?

Also, the user guide link seems to be not working any more.

Thanks!

**Comment by avocadosalsa** [comment]
You are correct, the fields are missing.  When the gexf file is imported, Cytoscape generates new a SUID for each `node` (similar in concept to the gexf `id`).  Because of this, when each `edge` is parsed, the original `source` and `target` attribute are only temporarily used to find a mapping between the old `node` `id` and new `SUID`.

Would it be better to add the gexf `node` `id` to the Node Table and add the gexf `edge` `source` and `target` to the Edge Table (using the original gexf values)
-OR-
add the mapped Cytoscape `node` `SUID` to the Edge Table for the appropriate `source` and `target` fields?

You should be able to get the user guide from the wiki here on GitHub.  https://github.com/bdemchak/gexf-app/wiki/GEXF-User-Guide.pdf
